test/extended/syncer/syncer.go

```diff
@@ -36,7 +36,7 @@ var _ = g.Describe("[area/transparent-multi-cluster]", func() {
 		defer k.AsPClusterKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("-f", mySyncer.OutputFilePath).Execute()
 		err := k.AsPClusterKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("apply").Args("-f", mySyncer.OutputFilePath).Execute()
 		o.Expect(err).ShouldNot(o.HaveOccurred())
-		mySyncer.WaitUntilReady(k)
+		mySyncer.WaitUntilReadyToUse(k)
```

Comment:
It is better to not define WaitUntilReadyToUse. Just call the waiting of the specific tested-api sync. Because currently we test deployment api, in future we may test more apis and different apis? And TBH in normal mind, it is hard to differentiate "Ready" VS "ReadyToUse" literally.
```suggestion
		mySyncer.WaitDeploymentsAPISynced(k)
```

Reply:
Addressed. I uses `WaitUntilReadyAndDeploymentsAPISynced` .